crawler_config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "base_urls": [],
-    "total_articles_to_find_and_parse": 0,
-    "max_number_articles_to_get_from_one_seed": 0
+    "base_urls": ["http://pravdapskov.ru/news?page=1", "http://pravdapskov.ru/news?page=2", "http://pravdapskov.ru/news?page=3", "http://pravdapskov.ru/news?page=4"],
+    "total_articles_to_find_and_parse": 100,
+    "max_number_articles_to_get_from_one_seed": 25
```

Comment:
you may implement logic for this param later. Just focus on implementing crawler main logic for now